ministryofjustice/analytics-platform-control-panel-frontend
# "What's new?" page

## What

Implement a new page at `/whats-new` which pulls in [markdown from the platform guidance repo](https://raw.githubusercontent.com/moj-analytical-services/platform_user_guidance/master/changelog/whats_new.md) and displays it. Also display a notification icon in the header bar when there are unseen notifications.

## How to review

1. Check out and see that there is a grey '?' icon in the top right of the header bar on, for example, the homepage. This indicates there is something new on the "What's new?" page that you haven't seen
2. Click the icon to visit the "What's new?" page – the icon will disappear, as you have now seen whatever it is that is new, and a cookie is created containing a hash of the markdown content
3. Go back to the homepage or to any other page, the icon should not display. Next time the what's new page is changed, the icon will reappear until the page is visited again and the cookie is updated
